CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ option (RPM "Generate RPM using CPack" OFF)
 option (NPM "Generate NPM/GYP tarballs" OFF)
 option (BUILDTESTS "Generate check-ups for upm" OFF)
 option (WERROR "Make all warnings into errors." ON)
+option (WUNUSED "Remove warnings if return value unused" OFF)
 
```

Comment:
Maybe the text: "Enable compiler warnings regarding unused return values." 

would be clearer...?

Reply:
I see - I was trying to remove the negative logic...  Here you set the default value to OFF.  Which means WUNUSED == OFF, and then the code below...